packages/core/src/components/GlobalActions/GlobalActions.stories.tsx

```diff
@@ -183,3 +201,113 @@ export const SectionVariant: StoryObj<HvGlobalActionsProps> = {
     );
   },
 };
+
+export const Breadcrumb: StoryObj<HvGlobalActionsProps> = {
```

Comment:
According to our guidelines:

> Avoid adding too many similar stories; each story should have a unique purpose.

How do these `Breadcrumb` and `Tabs` relate to `GlobalActions`? If the goal is to show how users can pass "custom content" to GlobalActions, then the two stories are redundant and we should merge them


And to that effect, we should probably disable Applitools too